src/screens/Platform/App.js

```diff
@@ -1,27 +1,32 @@
+// @flow
+
 import React from "react";
 import { usePlatformApp } from "@ledgerhq/live-common/lib/platform/PlatformAppProvider";
-
+import type { StackScreenProps } from "@react-navigation/stack";
 import { useTheme } from "@react-navigation/native";
 import TrackScreen from "../../analytics/TrackScreen";
 import WebPlatformPlayer from "../../components/WebPlatformPlayer";
 
-const PlatformApp = ({ route }: { route: { params: Props } }) => {
+const PlatformApp = ({ route }: StackScreenProps) => {
   const { dark } = useTheme();
-  const appId = route.params.platform;
+  const { platform: appId, ...params } = route.params;
   const { manifests } = usePlatformApp();
   const manifest = manifests.get(appId);
   const themeType = dark ? "dark" : "light";
 
   return (
-    <>
-      <TrackScreen category="Platform" name="App" />
-      <WebPlatformPlayer
-        manifest={manifest}
-        inputs={{
-          theme: themeType,
-        }}
-      />
-    </>
+    manifest && (
```

Comment:
return `{ manifest ? <>...</> : null }`
if manifest is ever falsy without being `null` it will create a react error where the component isnt returning something valid.

Reply:
Done 👌